src/MainUpdate.elm

```diff
@@ -818,6 +822,24 @@ update msg model =
                 in
                     ( newModel, Cmd.none )
 
+            MoveColumnUp columnIndex ->
+                let
+                    newPickedColumns =
+                        model.pickedColumns
+                            |> List.Extra.swapAt columnIndex (columnIndex - 1)
+                            |> Maybe.withDefault []
```

Comment:
here and below it could be safer to default to `model.pickedColumns` but happy to proceed as moving out of range is blocked by UI